README.md

```diff
@@ -2,13 +2,19 @@
 
 o-buttons provides Sass mixins and variables to create buttons.
 
+- [Usage](#usage)
 - [Markup](#markup)
 - [Sass](#sass)
 - [References](#references)
 - [Migration](#migration)
 - [Contact](#contact)
 - [Licence](#licence)
 
+## Usage
+
+There are two ways to include `o-buttons` in your project, consistent with other Origami components. See [how
+to include Origami components in your project](https://origami.ft.com/docs/components/#including-origami-components-in-your-project).
```

Comment:
I think the first sentence might need another draft, it feels a little gummy. I'm not sure how to improve it though. Maybe remove it, be more conversational?

```suggestion
Check out [how to include Origami components in your project](https://origami.ft.com/docs/components/#including-origami-components-in-your-project).
```

Reply:
Hmm it is gummy. I feel your suggestion is better but maybe a little abrupt, might if I append "to get started with `o-buttons`"?

```suggestion
Check out [how to include Origami components in your project](https://origami.ft.com/docs/components/#including-origami-components-in-your-project) to get started with `o-buttons`.
```

Reply:
ooh that's good i think, i like that a lot